doc/source/how-to/contributing.rst

```diff
@@ -0,0 +1,250 @@
+Contributing
```

Comment:
There is a contributing topic in the dev guide that explains how to contribute overall to a PyAnsys project. Each project has a contributing topic that refers to the dev guide topic and then supplies project-specific contribution information. I don't think we want to name this section Contributing. It would be confusing. Perhaps "General Coding Paradigms" to match the first sentence?

Reply:
Let us keep this title name for the moment. PyAnsys libraries should point to this guideline an expand it locally in their own CONTRIBUTING files.